scripts/update_schemastore.py

```diff
@@ -23,7 +23,7 @@
 
 def update_schemastore(schemastore: Path) -> None:
     if not schemastore.is_dir():
-        check_call(["git", "clone", schemastore_fork, schemastore])
+        check_call(["git", "clone", schemastore_fork, schemastore, "--depth=1"])
```

Comment:
Can we still do commits with this? (I'm not familiar enough with the git internals)

Reply:
Yup, I checked locally, it still works fine